src/main/java/life/qbic/openbis/openbisclient/OpenBisClient.java

```diff
@@ -1051,6 +1055,15 @@ public Vocabulary getVocabulary(String vocabularyCode) {
     }
   }
 
+  @Override
+  public void ingest(String dss, String serviceName, Map<String, Object> params) {
+    ServiceFinder serviceFinder2 =
```

Comment:
NIT: why this name? How about just serviceFinder or openBisServiceFinder?

Reply:
because it is the name that it was before (it was just commented out)